lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -97,11 +97,13 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   m_hasTs = (stageMask & (shaderStageToMask(ShaderStageTessControl) | shaderStageToMask(ShaderStageTessEval))) != 0;
   m_hasGs = (stageMask & shaderStageToMask(ShaderStageGeometry)) != 0;
 
-  SmallVector<Function *, 16> candidateCallees;
+  SmallVector<Function *, 16> inputCallees, outputCallees;
```

Comment:
For clearness, I suggest we add otherCallees because sendmsg is not a call to handle output. So I have

```
if ()
  inputCallees.push_back()
else if ()
  outputCallees.push_back()
else if ()
  otherCallees.push_back()
```

Reply:
Since we do not need to differentiate between output and sendmsg functions now. I would like to keep two SmallVector for simplicity. The inputCallees and OtherCallees. Does this sound ok to you?

Reply:
It sounds good. We can have just two categories: input and other.